internal/templates/common/yamls/service.yaml

```diff
@@ -11,15 +11,15 @@ metadata:
     {{ $.Values.app.group }}/app-deployment-version: {{ $deployment.version | quote }}
     {{ $.Values.app.group }}/is-isolated-run: "false"
     {{- range $i, $label := $deployment.labels }}
-    {{ $label.name }}: {{ $label.value }}
+    {{ $label.name }}: "{{ $label.value }}"
     {{- end }}
     {{- range $k, $v := $process.extra.serviceMetadata.labels }}
-    {{ $k }}: {{ $v }}
+    {{ $k }}: "{{ $v }}"
```

Comment:
would it be possible to use helm pipeline function here?

https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#pipelines

```
........ | quote }}
```

Reply:
same inside everywhere we used hardcoded " "?

Reply:
the resulting yamls are valid for kubernetes, helm validates the `metadata` section and that's the issue.